docs-beta/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@lage/docs-beta",
```

Comment:
let's change this to @lage-run/docs-beta actually

Reply:
Yes, this is the name of this package, it shouldn't be visible on the site itself I wouldn't think, and it is a good convention to name them all as part of the lage branding...